src/Auth0.OidcClient.MAUI.Platforms.Windows/Helpers.cs

```diff
@@ -0,0 +1,101 @@
+
+        /// <summary>
+        /// Helper property to verify the application is packaged.
+        /// </summary>
+        /// <remarks>
+        /// Original source: https://github.com/dotMorten/WinUIEx
+        /// </remarks>
+        /// <returns>A boolean indicate whether or not the app is packaged.</returns>
+        public bool IsAppPackaged
+        {
+            get
+            {
+                try
+                {
+                    // Application is MSIX packaged if it has an identity: https://learn.microsoft.com/en-us/windows/msix/detect-package-identity
+                    int length = 0;
+                    var sb = new StringBuilder(0);
+                    int result = GetCurrentPackageFullName(ref length, sb);
+                    return result != AppModelErrorNoPackage;
+                }
+                catch
+                {
+                    return false;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Helper method to verify the scheme is defined as a protocol in the AppxManifest.xml files
+        /// </summary>
+        /// <remarks>
+        /// Original source: https://github.com/dotMorten/WinUIEx
+        /// </remarks>
+        /// <param name="scheme">The scheme expected to be declared.</param>
+        /// <returns>A boolean indicate whether or not the scheme is declared as an Uri protocol.</returns>
+        public bool IsUriProtocolDeclared(string scheme)
+        {
+            if (global::Windows.ApplicationModel.Package.Current is null)
+                return false;
+            var docPath = Path.Combine(global::Windows.ApplicationModel.Package.Current.InstalledLocation.Path, "AppxManifest.xml");
+            var doc = XDocument.Load(docPath, LoadOptions.None);
+            var reader = doc.CreateReader();
+            var namespaceManager = new XmlNamespaceManager(reader.NameTable);
+            namespaceManager.AddNamespace("x", "http://schemas.microsoft.com/appx/manifest/foundation/windows10");
+            namespaceManager.AddNamespace("uap", "http://schemas.microsoft.com/appx/manifest/uap/windows10");
+
+            // Check if the protocol was declared
+            var decl = doc.Root?.XPathSelectElements($"//uap:Extension[@Category='windows.protocol']/uap:Protocol[@Name='{scheme}']", namespaceManager);
+
+            return decl != null && decl.Any();
+        }
+
+        /// <summary>
+        /// Helper method to open the browser through the url.dll.
+        /// </summary>
+        /// <param name="uri">The Uri to open</param>
+        public void OpenBrowser(Uri uri)
+        {
+            var process = new System.Diagnostics.Process();
+            process.StartInfo.FileName = "rundll32.exe";
+            process.StartInfo.Arguments = $"url.dll,FileProtocolHandler \"{uri.ToString().Replace("\"", "%22")}\"";
+            process.StartInfo.UseShellExecute = true;
+            process.Start();
+        }
```

Comment:
Before going to GA, we will want to have a decent security review on this. It works as expected, and is also used in other packages, but does look a bit funky.

If this turns out to be an issue before we go GA, we can swap for an embedded webview implementation.